duo_client/client.py

```diff
@@ -328,6 +328,14 @@ def _make_request(self, method, uri, body, headers):
     def _disconnect(self, conn):
         conn.close()
 
+    def normalize_paging_args(self, limit=None, offset=0):
```

Comment:
Could you add a docstring for this method?

Reply:
Good call.